codec_specs.md

```diff
@@ -58,6 +58,18 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
 
 A `Codec Mapping` MAY only be defined with a `Superseded By` value, if it has an expressed `Deprecation Date`. If used, the `Superseded By` value MUST store the `Codec ID` of another `Codec Mapping` that has superseded the `Codec Mapping`.
 
+## Recommendations for the Creation of New Codec Mappings
+
+Creators of new Codec Mappins to be used in the context of Matroska:
```

Comment:
Backtick all special terms just as above: `Codec Mapping`, `Codec ID` and `Codec Name`